app/frontend/src/resources/CouchersLogo.tsx

```diff
@@ -18,13 +19,20 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function CouchersLogo(props: SvgIconProps) {
+export interface CouchersLogoProps {
```

Comment:
```suggestion
export interface CouchersLogoProps extends SvgIconProps {
```
For that sweet autocomplete

Reply:
Actually, if the extra props aren't used anywhere, better not include them at all and get rid of the `{...SVgIconProps}` below